CHANGELOG.md

```diff
@@ -7,6 +7,27 @@ complete changelog, see the git history for each version via the version links.
 
 [hex package page]: https://hex.pm/packages/ex_machina
 
+## [2.1.0]
+
+### Breaking changes
+
+- Supports only Ecto 2.2 or higher https://github.com/thoughtbot/ex_machina/commit/8f494f1b72612ac4784f801e0f658ce30b4b3552
```

Comment:
I believe this means we need to update to `3.0.0`. Also, I believe there's another breaking change, but I can't remember what it was. @germsvel probably knows more.

Reply:
Yes, there was another breaking change that I was hoping to fix but haven't had the time. Someone actually opened an issue about it https://github.com/thoughtbot/ex_machina/issues/243

Reply:
I don't know if only supporting Ecto 2.2 means that we should bump to 3.0., unless it breaks ex_machina's public api. This is from [semver.org](http://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api):

> What should I do if I update my own dependencies without changing the public API?

> That would be considered compatible since it does not affect the public API. Software that explicitly depends on the same dependencies as your package should have their own dependency specifications and the author will notice any conflicts.

Reply:
@jsteiner I also just remembered that the PR I merged maintains support for Ecto 2.1. So it's actually not a breaking change in this release. So I think 2.1.0 works